src/layouts/Settings.tsx

```diff
@@ -28,7 +32,7 @@ const Settings = () => {
       </PageHeader>
       <BasicTabs
         tabs={tabsOptions}
-        value={pathname === SETTINGS_ROUTE ? VAT_RATE_ROUTE : pathname}
+        value={pathname === SETTINGS_ROUTE ? ORGANIZATION_INFORMATIONS_ROUTE : pathname}
```

Comment:
In case this changes in the future, we should probably change it to `value={pathname === SETTINGS_ROUTE ? tabsOptions[0]?.key : pathname}`

Reply:
Absolutely ! Updated